Makefile

```diff
@@ -100,6 +100,7 @@ compile-thunk:
 		--inject:${ROOTDIR}/testing/injections/buffer_shim.js \
 		--format=iife \
 		--define:process.env.IN_ISOLATED_VM_OR_BROWSER=true \
+		--define:process.env.NODE_DEBUG=false \
```

Comment:
can you add a note in code why this is added?

Reply:
Added a comment mentioning that util.format depends on debuglog which depends on NODE_DEBUG, with a link to the source code at https://github.com/nodejs/node/blob/6b055f385744d2ca71c19d46a0ec3bcfc51f5cd3/lib/internal/util/debuglog.js#L21

Reply:
ah, got it. that makes sense.